gluster_exporter/metric_brick.go

```diff
@@ -167,8 +190,18 @@ func BrickUtilization() {
 					glusterBrickInodesTotal.With(lbls).Set(usage.InodesAll)
 					glusterBrickInodesFree.With(lbls).Set(usage.InodesFree)
 					glusterBrickInodesUsed.With(lbls).Set(usage.InodesUsed)
+					if usage.Used >= maxBrickUsed {
```

Comment:
skip exporting if `brick.Type == "Arbiter"`. Reason: If both data bricks are down then Arbiter brick exposes less size and overall volume utilization will be shown wrong. Instead of that we can skip exporting if the brick is arbiter brick. With this approach, overall volume utilization will not contain the subvolume utilization so that it can be used as alerting in Prometheus.

Reply:
Ack

Reply:
`if brick.Type != "Arbiter" && usage.Used >= maxBrickUsed`

Reply:
and code comment can be "Skip exporting utilization data in case of arbiter brick to avoid wrong values when both the data bricks are down"

Reply:
Ack